pkg/apis/seedmanagement/validation/managedseed.go

```diff
@@ -296,3 +305,31 @@ func validateGardenClientConnection(gcc *config.GardenClientConnection, bootstra
 
 	return allErrs
 }
+
+func validateManagedSeedOperation(operation string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if operation == "" {
+		return allErrs
+	}
+
+	if operation != "" && !availableManagedSeedOperations.Has(operation) {
+		allErrs = append(allErrs, field.NotSupported(fldPath, operation, sets.List(availableManagedSeedOperations)))
+	}
+
+	return allErrs
+}
+
+func validateManagedSeedOperationUpdate(newOperation, oldOperation string, fldPath *field.Path) field.ErrorList {
```

Comment:
same here

Reply:
The issue from the seed cannot happen for the managed seed, because there is only `renew-kubeconfig` annotation which is not removed immediately...until we introduce a new annotation.